FrontEnd/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_BASE_API_URL=http://localhost:8000
```

Comment:
nit: `.env` should be ignored by git. I think its covered here https://github.com/ita-social-projects/Forum/pull/225/files

Reply:
Fixed